### PR TITLE
Restore cross-validation sanity test

### PR DESCRIPTION
**Pull request by lantz**
This is an attempt to restore running the cross-validation test as part of github actions.

I think I inadvertently deleted it when refactoring github actions to split out individual test phases and results; the cross validation test itself seems to have broken when we renamed the repo.

### CODE REVIEW

**Comment by lantz** [comment]
@juraul1  I'm probably doing this wrong.

**Comment by juraul1** [comment]
@lantz I'm not really familiar with `dirname $0` but it was returning `.` hence issues later. Not sure `pwd` is the best option but at least it's working and tests are passing.

**Comment by lantz** [comment]
Haha, indeed. I though I tried that actually and I'm shocked it works, but great!

File: cross-validation-tests/cross_validation.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-SCRIPT_DIR=$(dirname $0)
+SCRIPT_DIR=$(pwd)
```
Comment (lantz) [inline cross-validation-tests/cross_validation.sh:3]: 👍

File: cross-validation-tests/cross_validation.sh [hunk 2]
```diff
@@ -137,7 +137,7 @@ PYTHONPATH=.. $PYTHON auto_tests.py
 
 
 ###################### Cross-validation analysis - start ######################
-cd ../$CROSS_VAL_DIR
+cd $CROSS_VAL_DIR
```
Comment (lantz) [inline cross-validation-tests/cross_validation.sh:140]: 👍

**Comment by lantz** [comment]
I think I'll leave `bash -x` for now in case any issues arise in the future.

**Comment by lantz** [comment]
closes #134

**Pull request merged**